libs/common-utilities/src/utils/ipfs.ts

```diff
@@ -0,0 +1,24 @@
+export const PINATA_PIN_FILE_API =
+  'https://api.pinata.cloud/pinning/pinFileToIPFS';
+
+export const pinataIpfsPost = async ({
+  creds,
+  formData,
+}: {
+  creds: { pinata_api_key: string; pinata_api_secret: string };
+  formData: FormData;
+}) => {
+  try {
+    const response = await fetch(PINATA_PIN_FILE_API, {
+      method: 'POST',
+      headers: {
+        pinata_api_key: creds.pinata_api_key,
+        pinata_secret_api_key: creds.pinata_api_secret,
```

Comment:
How do devs get access to creds?

Reply:
devs need to pass their own creds

Reply:
For sure. What do we have in mind for comm devs to be able to have the own creds, existing env var process (not necessarily opposed to this.